src/golang/lib/workflow/operator/operator.go

```diff
@@ -114,6 +123,12 @@ func NewOperator(
 		db:                  db,
 
 		execMode: execMode,
+		execState: shared.ExecutionState{
```

Comment:
Shouldn't we use the time when the `OperatorResult` is initialized in the database as the time when the operator result is set to `PendingAt`?

Reply:
I actually feel removing the dependency on DB is more important here. For example, a preview can also have a `pending_at` and this clearly has nothing to do with DB. So here I think using the object's in-memory creation time is sufficiently good